api/tm-forum/customer-bill-management/api.json

```diff
@@ -512,6 +512,70 @@
                         }
                     }
                 }
+            },
```

Comment:
Please dont change this file, its the original version as published by TMForum. The changes should go into the extension apis: https://github.com/FIWARE/tmforum-api/tree/main/customer-bill-management/api